caffe2/python/helpers/conv.py

```diff
@@ -54,20 +54,23 @@ def _ConvBase(
         weight_shape.extend(kernels)
         weight_shape.append(int(dim_in / group))
 
+    weight_initializer = initializers.update_initializer(
+        WeightInitializer, weight_init, ("XavierFill", {})
+    )
+    bias_initializer = initializers.update_initializer(
+        BiasInitializer, bias_init, ("ConstantFill", {})
+    )
+
     if model.init_params:
-        weight = model.param_init_net.__getattr__(weight_init[0])(
-            [],
-            blob_out + '_w',
+        weight = model.create_param(
```

Comment:
This unfortunately breaks things, as now weight & bias are added twice to params.

Test python/data_parallel_model_test.py -- test_checkpoint_params will fail.

I am uncommiting this for now.